aristamd/src/app/components/accordion/accordion.component.js

```diff
@@ -0,0 +1,54 @@
+
+var accordion = {
+  template: require('./accordion.html'),
+  controller: AccordionController,
+  bindings: {
+    headerContent: '<',
+    bodyContent: '<',
+    id:'@',
+  }
+}
+
+AccordionController.$inject = ['$timeout', 'communicationCenterService']
+
+function AccordionController ($timeout, communicationCenterService) {
+  var ctrl = this;
+  var latestClass = 'fa fa-chevron-right';
+
+  ctrl.$onInit = function () {
+      if (angular.isUndefined(ctrl.id)) {
+        throw('AristaMD Accordion Error: accordion needs an id');
+      }
+
+      ctrl.isOpen = false;
+      ctrl.accordionClass = 'fa fa-chevron-right';
+      ctrl.accordionChannel = communicationCenterService.createChannel(ctrl.id);
+
+      ctrl.accordionChannel.on('onToggleAccordion')
+      .subscribe(function (forceState) {
+        ctrl.toggleAccordion(forceState);
+      });
+
+      ctrl.toggleAccordion = function (forceState) {
+        if (!angular.isUndefined(forceState)) {
+          ctrl.accordionClass = forceState === true ? 'fa fa-chevron-down' : 'fa fa-chevron-right';
```

Comment:
explicit boolean comparison may don't needed. Should we use this as ng-class instead?